### ISSUE TITLE
Applying augmentations - any ideas?

### ISSUE THREAD

**Issue by SahPet** [prov-gigapath/prov-gigapath#29]
I've tested finetuning of the model successfully for some epochs now, but I'm having trouble with overfitting (train loss decreasing, val loss increasing). Since the embeddings are pre-made applying geometric augmentations like flip/rotate etc is not very straight forward. Does anyone have any ideas on this? Applying augmentations during the embedding creation phase? Seems like one would get a huge amount (e.g. 8x if all flip/rotate combinations are applied) of embeddings data in that case.
